docs/considerations.rst

```diff
@@ -0,0 +1,267 @@
+years, it is likely that more powerful approaches to multi-echo denoising will sufficiently mature and add
+even more value to a data set.
+
+Other multi-echo denoising methods, such as MEICA, the predecessor to tedana, have shown the potential for
+much greater data quality improvements, as well as the ability to more accurately separate visually similar
+signal vs noise, such as scanner based drifts vs slow changes in BOLD signal. 
+These more powerful methods are
+still being improved, and the algorithms are still changing. 
+Users need to have the time and knowledge to look
+at the denoising output from every run to make sure denoising worked as intended. 
+
+You may recover signal in areas affected by dropout
+---------------------------------------------------
+Typical single echo fMRI uses an echo time that is appropriate for signal across most of the brain. 
+While this is effective
+it also leads to drop out in regions with low :math:T_2^* values. 
+This can lead to low or even no signal at all in some areas. 
+If your research question could benefit from having either
+improved signal characteristics in regions such as the orbitofrontal cortex, ventral temporal cortex or 
+the ventral striatum them multi-echo fMRI may be beneficial. 
+
+Consider the cost of added quality control
+------------------------------------------
+The developers of ``tedana`` strongly support always examining data for quality concerns, whether
+or not multi-echo fMRI is used. 
+If someone wants a push-button
+way to use multi-echo data to improve data quality, that doesn't require as deep an inspection of every output,
+stick with using the weighted average. 
+The developers of tedana look forward to when tedana and other methods
+have sufficiently stable algorithms, which have been validated on a wide range of data sets, so that we can
+recommend the wide use of tedana.
+``tedana`` currently allows the use of a ``--png`` flag when called, in order to produce basic diagnostic images, 
+`see outputs`_ for more information on these outputs. 
+
+.. _t2smap: https://tedana.readthedocs.io/en/latest/usage.html#run-t2smap
+.. _see outputs: https://tedana.readthedocs.io/en/latest/outputs.html
+
+Acquisition Parameter Recommendations
+=====================================
+There is no empirically tested best parameter set for multi-echo acquisition.
+The guidelines for optimizing parameters are similar to single-echo fMRI.
+For multi-echo fMRI, the same factors that may guide priorities for single echo
+fMRI sequences are also relevant.
+Choose sequence parameters that meet the priorities of a study with regards to spatial resolution,
+spatial coverage, sample rate, signal-to-noise ratio, signal drop-out, distortion, and artifacts.
+
+A minimum of 3 echoes is required for running the current implementation fo TE-dependent denoising in
+``tedana``. 
+While there are successful studies that don’t follow this rule,
```

Comment:
Can we invert this sentence ? I'm still a little confused on first read-through, personally :confused:

Reply:
I've removed the not follow language - and just say we need three echoes, but left in the note that dual echo is not what we are talking about here.